<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CraftParam</class>
 <widget class="QWidget" name="CraftParam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="8" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>stay:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDoubleSpinBox" name="blow_time_">
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>gas:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>BAR</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="ratio_"/>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="frequency_"/>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="progressive_time_">
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>focus:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>jet h:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="pressure_"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>freq:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="enable_blow_">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>blow:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="gas_type_">
       <item>
        <property name="text">
         <string>air</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>oxygen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>nitrogen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>high air</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>high oxygen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>high nitrogen</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="focus_"/>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="enable_progressive_">
         <property name="text">
          <string>incr t:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Hz</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="height_"/>
     </item>
     <item row="9" column="2">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="power_">
       <property name="wrapping">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>pressure:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>ratio:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>power:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="stay_time_">
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>horizontalSpacer_3</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>enable_progressive_</sender>
   <signal>toggled(bool)</signal>
   <receiver>progressive_time_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enable_blow_</sender>
   <signal>toggled(bool)</signal>
   <receiver>blow_time_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>progressive_time_</sender>
   <signal>editingFinished()</signal>
   <receiver>CraftParam</receiver>
   <slot>onDataUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>height_</sender>
   <signal>editingFinished()</signal>
   <receiver>CraftParam</receiver>
   <slot>onDataUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gas_type_</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CraftParam</receiver>
   <slot>onDataUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pressure_</sender>
   <signal>editingFinished()</signal>
   <receiver>CraftParam</receiver>
   <slot>onDataUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>power_</sender>
   <signal>editingFinished()</signal>
   <receiver>CraftParam</receiver>
   <slot>onDataUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ratio_</sender>
   <signal>editingFinished()</signal>
   <receiver>CraftParam</receiver>
   <slot>onDataUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>frequency_</sender>
   <signal>editingFinished()</signal>
   <receiver>CraftParam</receiver>
   <slot>onDataUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>focus_</sender>
   <signal>editingFinished()</signal>
   <receiver>CraftParam</receiver>
   <slot>onDataUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stay_time_</sender>
   <signal>editingFinished()</signal>
   <receiver>CraftParam</receiver>
   <slot>onDataUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blow_time_</sender>
   <signal>editingFinished()</signal>
   <receiver>CraftParam</receiver>
   <slot>onDataUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enable_progressive_</sender>
   <signal>toggled(bool)</signal>
   <receiver>CraftParam</receiver>
   <slot>onDataUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>-21</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enable_blow_</sender>
   <signal>toggled(bool)</signal>
   <receiver>CraftParam</receiver>
   <slot>onDataUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>-11</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onDataUpdate()</slot>
 </slots>
</ui>
